comms.js

```diff
@@ -37,8 +37,8 @@ Comms.prototype.write = function(msg) {
 	}
 };
 
-Comms.prototype.listen = function(port) {
-	var self = this, server = this.server = jot.createServer(port);
+Comms.prototype.listen = function(port, opts) {
+	var self = this, server = this.server = jot.createServer(opts || {}, port);
```

Comment:
Also, in case port is a number, don't we have to call `createServer` with the specific port so we can listen to it in L50?